docs/_source/conf.py

```diff
@@ -143,27 +171,34 @@
         }
     </style>
 """
+    + f"""
 
-autodoc_typehints = "description"
+.. raw:: html
 
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates"]
+    {getting_started_html}
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
+"""
+    + """
 
-if not os.getenv("READTHEDOCS"):
-    notfound_urls_prefix = None
-else:
-    lang, docs_version = (
-        os.getenv("READTHEDOCS_LANGUAGE", "en"),
-        os.getenv("READTHEDOCS_VERSION", "latest"),
-    )
+.. raw:: html
 
-    notfound_urls_prefix = f"/{lang}/{docs_version}/"
+    <div class="row">
+    <div class="col">
+    <a href="https://colab.research.google.com/github/argilla-io/argilla/tree/"""
+    + os.getenv("READTHEDOCS_VERSION", "main").replace("-", "/")
+    + """/docs/_source/{{ env.doc2path(env.docname, base=None) }}"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a></div>"""
```

Comment:
For Colab (on the branch rtd deployment) I'm getting the following error. It seems like blob is missing:

```
malformed GitHub path: missing 'blob' before branch name: argilla-io/argilla/tree/docs/open_in_colab_button/docs/_source/tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb
```